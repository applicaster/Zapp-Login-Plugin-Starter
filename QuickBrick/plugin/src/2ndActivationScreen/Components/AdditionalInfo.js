import React from 'react';
import { View, Text } from 'react-native';

function AdditionalInfo(props) {
  const {
    additionalInfo,
    additionalInfoStyle
  } = props;

  return (
    <View style={styles.bottomText}>
      <Text
        style={additionalInfoStyle}
        numberOfLines={2}
        ellipsizeMode="tail"
      >
        {additionalInfo}
      </Text>
    </View>
  );
}

const styles = {
  bottomText: {
    marginTop: 88
  }
};

export default AdditionalInfo;
