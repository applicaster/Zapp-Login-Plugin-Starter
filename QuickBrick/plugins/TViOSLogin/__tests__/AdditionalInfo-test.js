import React from 'react';
import renderer from 'react-test-renderer';
import AdditionalInfo from '../src/2ndActivationScreen/Components/AdditionalInfo';


describe('AdditionalInfo Component', () => {
  it('Matches the snapshot', () => {
    const additionalInfo = renderer
      .create(<AdditionalInfo />)
      .toJSON();
    expect(additionalInfo).toMatchSnapshot();
  });
});
