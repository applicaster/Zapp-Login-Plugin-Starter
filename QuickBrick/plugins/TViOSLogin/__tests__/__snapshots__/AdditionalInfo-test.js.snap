// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdditionalInfo Component Matches the snapshot 1`] = `
<View
  style={
    Object {
      "marginTop": 88,
    }
  }
>
  <Text
    ellipsizeMode="tail"
    numberOfLines={2}
  />
</View>
`;
